common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -3,24 +3,23 @@
 @import common.{Edition, LinkTo}
 @import views.support.RenderClasses
 
-<li class="navigation-group__item navigation-group__item--edition-picker">
+<li class="fly-in-item">
```

Comment:
I am not sure I fully get this name. To me this won't work if we try to use the same markup for desktop. What about just calling it `menu` or something more generic?